### DISCUSSION TITLE
Idea for small bundle size

### DISCUSSION THREAD

**Discussion by robertaboukhalil** [fathominfo/delphy-web#2] [Ideas]
Hello, I came across this tool recently, really nice work!

Random thought: While digging around, I noticed that delphy.js is not minified and that the delphy.wasm file is 20MB. I'm wondering whether you ran Emscripten with -g? If so, in my experience removing -g and adding -O2 or -O3 help reduce the size of the wasm & js files :)

**Comment by benfry** [comment]
Yes! Thanks for making a note of it. While we've been doing these pre-releases, we've been building with the debug info for the WASM side, and also not minifying the JS, so we can quickly track down various bugs as they come up. But now that more people are starting to actually use it, it's time to do release builds for the public version.

**Comment by benfry** [comment]
Created an issue for it here so that we can track it: https://github.com/fathominfo/delphy-web/issues/3
